rpc/client_test.go

```diff
@@ -224,6 +273,7 @@ func generateTestReport(testdataDir, s3Name, s3Prefix string, duplicateMetric bo
 		TaskID:    "taskID",
 		Mainline:  true,
 		Execution: 2,
+		Requester: "RepotrackerVersionRequester",
```

Comment:
Also went ahead and added this per the fallout of this convo: https://jira.mongodb.org/browse/WRITING-11201